src/pause/138CC0.c

```diff
@@ -332,7 +462,7 @@ void pause_badges_init(MenuPanel *panel) {
     panel->initialized = TRUE;
 }
 
-void pause_badges_handle_input(void) {
+void pause_badges_handle_input(MenuPanel* panel) {
```

Comment:
Why did you add an arg here? Is there a call to it that passes an arg?

Reply:
I'm pretty sure the `_handle_input` functions are pointed to by this field in the `MenuPanel` struct, which would imply that they all have the same signature:
```c
    /* 0x10 */ UNK_FUN_PTR(fpHandleInput);
```
`pause_stats_handle_input` has that argument, so presumably `pause_badges_handle_input` has it as well.

Reply:
Ahh yeah okay, that makes sense